src/hera.cpp

```diff
@@ -42,6 +42,12 @@ using namespace std;
 using namespace wasm;
 using namespace HeraVM;
 
+struct hera_instance : evm_instance {
+  bool fallback = false;
+
+  hera_instance() : evm_instance({EVM_ABI_VERSION, NULL, NULL, NULL}) {}
```

Comment:
More C++ish would be `nullptr`. But anyway, why not set the functions here? I.e. `{EVM_ABI_VERSION, evm_destroy, evm_execute, evm_set_option}`.

Also, I would name the function with `hera_` prefix, not `evm_` prefix...

Reply:
> why not set the functions here?

For that I'll need to move code around. And need to add the types as arguments.

> Also, I would name the function with hera_ prefix, not evm_ prefix...

Yeah we could do that too.

Reply:
I'm merging this now, can you create a PR to clean it up even more? (Needs to be merged for the fallback false support)